lib/tasks/update_friend_volunteer_associations.rake

```diff
@@ -0,0 +1,5 @@
+desc 'Delete friend-volunteer associations that are >= 3 months old'
+task :update_friend_volunteer_associations => :environment do
+  old_relations = UserFriendAssociation.where('created_at < ?', 3.month.from_now)
```

Comment:
I think we want 3.months.ago, rather than 'from_now', since 'ago' is the past and 'from_now' is the future.  Also, cool thing, you can use 'month' and 'months' interchangeably, allowing you to be grammatically correct, thanks Ruby :)

Reply:
Ah, just another reason why I love Ruby as well. Yes, good call. I'll change it to `ago`.

Reply:
So in that case, we'd want to change the language to `'created_at > ?', 3.months.ago` because we are evaluating data against a date in the past, correct?

Reply:
It should stay 'created_at < ?', I believe.  If created_at is 4.months.ago, it will be considered an 'old_relation'.  If created_at is 2.months.ago, it will not be considered an 'old_relation'.  This is what we want.  Does that make sense?

Reply:
@benhayehudi this is one of two outstanding changes, everything else is looking great.  See comment above about  'created_at < ?', which was changed to 'created_at > ?' in a commit after this.  Everything else is looking good!